docs/providers/aws/cli-reference/invoke.md

```diff
@@ -31,6 +31,21 @@ Default is `false`.
 ## Provided lifecycle events
 - `invoke:invoke`
 
+
+# Invoke Local
+
+Invokes a function locally for testing and logs the output.
+
+```bash
+serverless invoke local --function functionName
+```
+
+## Options
+- `--function` or `-f` The name of the function in your service that you want to invoke locally. **Required**.
+- `--path` or `-p` The path to a json file holding input data to be passed to the invoked function. This path is relative to the
+root directory of the service. The json file should have an `event` and `context` properties to hold your mocked event and context data.
```

Comment:
"Should have `event` and `context` properties" or "Should have an `event` and `context` property".